addons/mail/static/src/models/message_notification_view.js

```diff
@@ -0,0 +1,41 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
+
+const { markup } = owl;
+
+registerModel({
+    name: 'NotificationMessageView',
+    recordMethods: {
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeMessage() {
+            if (this.messageListViewItemOwner) {
+                return this.messageListViewItemOwner.message;
+            }
+            return clear();
+        },
+        _computeNotificationText() {
+            return markup(`<span class="d-inline">` + this.message.authorName + `</span> ` + this.message.prettyBody);
+        }
```

Comment:
Why not just put it in template?

Also it's best practice to not deal with sibling text nodes, as it's annoying in css for styling purpose (e.g. text-ellipsis does not behave/work correctly), and instead wrap all text nodes in a html node e.g. `<span>`.
So best have a `<span>` also for the `prettyBody` part.